tests/Unit/Domain/CoordinateMaps/Test_EquatorialCompression.cpp

```diff
@@ -0,0 +1,87 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cmath>
+#include <random>
+
+#include "Domain/CoordinateMaps/EquatorialCompression.hpp"
+#include "ErrorHandling/Error.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/StdArrayHelpers.hpp"
+#include "tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.EquatorialCompression",
+                  "[Domain][Unit]") {
+  // Set up random number generator
+  std::random_device rd;
+  std::mt19937 gen(rd());
+  std::uniform_real_distribution<> aspect_ratio_dis(0.1, 10);
+
+  const double aspect_ratio = aspect_ratio_dis(gen);
+  CAPTURE_PRECISE(aspect_ratio);
+  const CoordinateMaps::EquatorialCompression angular_compression_map(
+      aspect_ratio);
+  test_suite_for_map(angular_compression_map);
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.EquatorialCompression.Radius",
+                  "[Domain][Unit]") {
+  // Set up random number generator:
+  const auto seed = std::random_device{}();
+  std::mt19937 gen(seed);
+  INFO("seed = " << seed);
+  std::uniform_real_distribution<> real_dis(-10, 10);
+  std::uniform_real_distribution<> aspect_ratio_dis(0.1, 10);
+  const double x = real_dis(gen);
+  CAPTURE_PRECISE(x);
+  const double y = real_dis(gen);
+  CAPTURE_PRECISE(y);
+  const double z = real_dis(gen);
+  CAPTURE_PRECISE(z);
+  const double aspect_ratio = aspect_ratio_dis(gen);
+  CAPTURE_PRECISE(aspect_ratio);
+  const std::array<double, 3> input_point{{x, y, z}};
+  const CoordinateMaps::EquatorialCompression angular_compression_map(
+      aspect_ratio);
+  const auto result_point = angular_compression_map(input_point);
+  const auto& result_x = result_point[0];
+  const auto& result_y = result_point[1];
+  const auto& result_z = result_point[2];
+
+  // The EquatorialCompression map should preserve radial lengths.
+  CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+
+  // It is easy to mix up \alpha with one over \alpha,
+  // so this comment serves as a reminder to future readers
+  // as to which direction the \alpha should go in the
+  // CHECK statement.
+  //
+  // The aspect ratio is the ratio of the horizontal length
+  // to the vertical height. Conveniently, the aspect ratio is
+  // equivalent to tan \theta for \theta the polar angle.
```

Comment:
I'm still confused by this sentence, even after your clarification; in what sense is the aspect ratio "equivalent" to tan(theta)? They're not numerically equal... do you mean what's said in the equation below (in which case is this sentence even needed?)? or something else?

Reply:
Hi Francois, I've given it more thought and I think you have a point. This choice of wording came from the fact that points with tan theta = 1 get mapped to points with tan theta' = alpha. From there the intuition is that increasing alpha increases the resulting tan theta' of the mapped points. From that and the fact that both tan theta and the aspect ratio (for a given point) are given by sqrt(x^2+y^2)/z I said that the aspect ratio is tan theta. However this is inconsistent with thinking about a single `aspect_ratio` argument that represents the global distortion of the entire map - in this case the constant `aspect_ratio` is certainly not equal to tan theta. I agree the wording needs changing.

I propose replacing the sentence with "The parameter name `aspect_ratio` is used because points with tan theta = 1 get mapped to points with tan theta' = aspect_ratio". The original purpose of this sentence was to provide a motivation for why the name "aspect_ratio" was chosen. Sorry for the confusion, thanks for pointing it out!